examples/export_runs.py

```diff
@@ -0,0 +1,52 @@
+from domino import Domino
+import pandas as pd
+import json
+import os
+
+output_dir = "results"
+
+# connect to domino; be sure to have these environment variables set
+#  (runs inside a Domino executor automatically set these for you)
+domino = Domino("nick/digits", api_key=os.environ['DOMINO_USER_API_KEY'], host=os.environ['DOMINO_API_HOST'])
```

Comment:
I was checking this. `nick/digits` is a private project so this probably needs to be an argument or something.